src/components/atoms/forms/form-item.scss

```diff
@@ -0,0 +1,60 @@
+@use "../../../sass/mixins";
+@use "../../../sass/settings";
+@use "../../../sass/typography";
+
+.form-item {
+  // @include blg-spacing("bottom", "small", "margin");
```

Comment:
We are not using blg-spacing on epp. We need to add margin 24px to the bottom of this.